utils/obsproc/Ghrsst2Ioda.h

```diff
@@ -184,7 +184,8 @@ namespace gdasapp {
         }
       }
       // Calculate the average and store datetime
-      int64_t seconds_s = static_cast<int64_t>(sum / count);
+      // Replace the seconds_s to o when count is zero
+      int64_t seconds_s = count != 0 ? static_cast<int64_t>(sum / count) : 0;
```

Comment:
sorry but I forgot, was this the cause of the failure when processing the obs in March?

Reply:
> sorry but I forgot, was this the cause of the failure when processing the obs in March?

Yes, it is- 5 out of 164 obs has no datetime, so count was 0-